net/newsboat/files/patch-Makefile.diff

```diff
@@ -6,21 +6,24 @@ The target "doc/example-config" is a dependency of "install-examples" only, so
 it runs in the destroot phase, add it to "all" to avoid that. 
 --- Makefile
 +++ Makefile
-@@ -18,7 +18,7 @@
+@@ -18,9 +18,9 @@
  DEFINES+=-DGIT_HASH=\"$(GIT_HASH)\"
  endif
  
 -WARNFLAGS=-Werror -Wall -Wextra -Wunreachable-code
 +WARNFLAGS=-Wall -Wextra -Wunreachable-code
  INCLUDES=-Iinclude -Istfl -Ifilter -I. -Irss
- BARE_CXXFLAGS=-std=c++11 -O2 -ggdb $(INCLUDES)
+-BARE_CXXFLAGS=-std=c++11 -O2 -ggdb $(INCLUDES)
++BARE_CXXFLAGS=-std=c++11 $(INCLUDES)
```

Comment:
May I ask why you disable optimizations?

Reply:
MacPorts automatically adds [-Os](https://clang.llvm.org/docs/CommandGuide/clang.html#code-generation-options) to the compiler flags, so overriding `-Os` with `-O2` is not desirable.

In MacPorts, some ports have a "debug" variant to add flags like `-g` or `-ggdb` to the compiler flags.
However, such flags are generally not added by default.

Reply:
> overriding -Os with -O2 is not desirable

If MacPorts does something like `export CXXFLAGS=-Os`, then everything will work fine out of the box: our Makefile appends `CXXFLAGS` at the very end of the compilation flags, so that environment variable always overrides whatever we set by default.

Got it re `-ggdb`. I don't plan to take it out, but if many other package/port maintainers find it useless, I'll reconsider.

Reply:
I see your point.
Thank you for pointing it out.
I have modified the pull request.